### PR TITLE
[POC] make grid command

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: config/services/_main.yaml [hunk 1]
```diff
@@ -20,6 +20,11 @@ services:
         # fetching services directly from the container via $container->get() won't work
         public: false
 
+        bind:
+            $projectDir: '%kernel.project_dir%'
```
Comment (loic425) [inline config/services/_main.yaml]: @lchrusciel It's here just to save time, but services arguments should be manually configured on the future bundle.

File: src/Maker/Helper/GridHelper.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Maker\Helper;
+
+class GridHelper
```
Comment (lchrusciel) [inline src/Maker/Helper/GridHelper.php]: I would still recommend `final` + `interface` on helper classes

**Comment by lchrusciel** [comment]
Or maybe grid and resource integration should be part of GridBundle and ResourceBundle? I would say, that it would be even better ;)

File: src/Maker/Helper/GridHelper.php (hunk 2 above)
Comment (loic425) [reply src/Maker/Helper/GridHelper.php]: yes exactly.

Comment (loic425) [reply src/Maker/Helper/GridHelper.php]: For the state, it's not publishable yet, I've not finished actions and yes filtration is missing.

**Comment by lchrusciel** [comment]
We can load these services only in dev environment anyway

**Comment by loic425** [comment]
This PR is now on SyliusGridBundle: https://github.com/Sylius/SyliusGridBundle/pull/46

**Pull request closed without merging**